lib/screens/splash/splash.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flutter/material.dart';
+import 'package:survey_flutter/gen/assets.gen.dart';
+
+class SplashScreen extends StatefulWidget {
+  const SplashScreen({Key? key}) : super(key: key);
+
+  @override
+  State<StatefulWidget> createState() => SplashScreenState();
+}
+
+class SplashScreenState extends State<SplashScreen> {
+  bool _isLogoVisible = false;
+  bool _shouldStartAnimation = false;
+
+  @override
+  void initState() {
+    super.initState();
+    _delayedLogoVisibility();
+  }
+
+  void _delayedLogoVisibility() async {
+    await Future.delayed(const Duration(seconds: 1));
+    setState(() {
+      _isLogoVisible = true;
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: LayoutBuilder(builder: (context, constraints) {
+        return Stack(
+          alignment: AlignmentDirectional.center,
+          fit: StackFit.expand,
+          children: [
+            Image.asset(
+              Assets.images.splashBackground.path,
+              fit: BoxFit.cover,
+            ),
+            _buildAnimatedLogo(constraints),
+          ],
+        );
+      }),
+    );
+  }
+
+  Widget _buildAnimatedLogo(BoxConstraints constraints) {
+    return AnimatedPositioned(
+      duration: const Duration(seconds: 1),
```

Comment:
Since we're using the duration with 1 second twice, it would be better to define a constant to use 🤔

Reply:
As I returned only `AnimatedOpacity` directly without wrapping in `AnimatedPositioned` at https://github.com/nimblehq/flutter-ic-khanh-thieu/pull/50/commits/a3e0891d340fcf6f2aa5ce76cbb92b2e479eb1a8 and let the login handle the moving up animation, so we might not need to define a constant here 🙏